docker-compose.yml

```diff
@@ -154,7 +154,7 @@ services:
     restart: on-failure
 
   browserless:
-    image: browserless/chrome:1.56.0-puppeteer-18.0.5@sha256:83a0f24eb4341c30e0f5106a98f1d1e5a3981805fce2829403cda9ef2e0b965e
+    image: browserless/chrome:1.56.0-puppeteer-18.0.5
```

Comment:
Isn't renovate going to immediately re-pin this?

Reply:
Hmm, don't know to be honest. Shall we try, and if renovate comes back we can reconfigure renovate?

Reply:
Sure, whatever. But maybe you can in the meantime already prepare by determining a SHA which works for you on mac? Then we can pin it to that tag for now.

Reply:
[07c3de3fe3578b1a3c3b9f3af5595c2a8f886fbc1c1d5c5cc3f9d2623a0f72ac](https://hub.docker.com/layers/browserless/chrome/1.56.0-puppeteer-18.0.5/images/sha256-07c3de3fe3578b1a3c3b9f3af5595c2a8f886fbc1c1d5c5cc3f9d2623a0f72ac?context=explore)

But that's linux/arm64, so that's not going to work for you (and also not for Windows-me 😄).